src/lib/data/near.js

```diff
@@ -42,10 +42,10 @@ const TestNearConfig = {
 };
 
 const CalimeroConfig = {
-  networkId: "testingshard-calimero-testnet",
-  calimeroUrl: "https://api.staging.calimero.network/api/v1/shards/testingshard-calimero-testnet/neard-rpc/",
+  networkId: `${process.env.NEXT_PUBLIC_SHARD_ID}`,
+  calimeroUrl:`https://api.staging.calimero.network/api/v1/shards/${process.env.NEXT_PUBLIC_SHARD_ID}/neard-rpc/`,
```

Comment:
Is better to pass the complete rpc Url so we can use the image for different environments

```suggestion
  calimeroUrl:`process.env.NEXT_PUBLIC_RPC_URL}`,
```